mllib/src/main/scala/org/apache/spark/mllib/optimization/LBFGS.scala

```diff
@@ -92,9 +92,11 @@ class LBFGS(private var gradient: Gradient, private var updater: Updater)
   }
 
   /**
+   * :: DeveloperApi ::
    * Set the gradient function (of the loss function of one single data example)
    * to be used for L-BFGS.
    */
+  @DeveloperApi
```

Comment:
The `LBFGS` class is marked `DeveloperApi`. Do we still need to mark its members?